lib/button_section.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:flutter/material.dart';
+import 'styles.dart';
+
+class ButtonSection extends StatefulWidget {
+  const ButtonSection({
+    super.key,
+  });
+
+  @override
+  State<ButtonSection> createState() => _ButtonSectionState();
+}
+
+class _ButtonSectionState extends State<ButtonSection> {
+  int _counter = 0;
+  static const Color buttonColor = Color(0xffAB0034);
+
+  static const double paddingBetweenCounterTextandButton = 15;
+  static const double likebuttonWidth = 50;
+  static const double likeButtonHeigth = 50;
+
+  void _incrementCounter() {
+    setState(() {
+      _counter++;
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Center(
+      child: Column(
+        children: [
+          Text(
+            "People who liked the movie: $_counter",
```

Comment:
you can move the hardcoded part of the text into his own constant and concatenate it to the counter

Reply:
Done